src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -121,32 +134,64 @@ class BaseCreateCourseForm extends React.Component {
             }
             required
           />
-          <Field
-            name="enrollmentTrack"
-            component={RenderSelectField}
-            options={this.getEnrollmentTrackOptions()}
-            label={
-              <FieldLabel
-                id="enrollment-track-label"
-                text="Enrollment track"
+          {usingCourseType ? (
+            <React.Fragment>
+              <Field
+                name="type"
+                component={RenderSelectField}
+                options={courseTypeOptions}
+                label={
+                  <FieldLabel
+                    id="course-type-label"
+                    text="Course Type"
+                    required
+                    helpText={(<div><p>TODO: Come up with type helpText</p></div>)}
+                  />
+                }
                 required
-                helpText={enrollmentHelp}
               />
-            }
-            required
-          />
-          {ENTITLEMENT_TRACKS.includes(currentFormValues.enrollmentTrack) && <Field
-            name="price"
-            component={RenderInputTextField}
-            extraInput={{
-              min: 1.00,
-              step: 0.01,
-              max: 10000.00,
-            }}
-            type="number"
-            label={<FieldLabel text="Price (USD)" required />}
-            required
-          />}
+              {entitlementUUIDS.includes(currentFormValues.type) && <Field
+                name="price"
+                component={RenderInputTextField}
+                extraInput={{
+                  min: 1.00,
+                  step: 0.01,
+                  max: 10000.00,
+                }}
+                type="number"
+                label={<FieldLabel text="Price (USD)" required />}
+                required
+              />}
+            </React.Fragment>) : (
+              <React.Fragment>
```

Comment:
Super-nit: different indentation levels for these two is odd to me.

Reply:
eslint :(